ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/activityBlock/activityBlock.component.ts

```diff
@@ -85,7 +85,9 @@ export class ActivityBlockComponent implements OnInit, OnDestroy {
     }
 
     private openCreatedInstanceDialog(instanceGuid: string): void {
-        this.modalActivities.find(activity => activity.instance.instanceGuid === instanceGuid)
-            .openEditDialog();
+        const modalActivity = this.modalActivities.find(activity => activity.instance.instanceGuid === instanceGuid);
```

Comment:
I think you can do something like:
this.modalActivities.find(activity => activity.instance.instanceGuid === instanceGuid)?.openEditDialog();